### PR TITLE
feat: add support for shadow DOM styling

### LINKED ISSUE #41: Cannot edit slider dots styles because CSS @apply has been removed
In your code you use properties:

* `--paper-slider-dot-container-styles` | (Mixin) Custom styles for dot container | NULL
* `--paper-slide-dot-styles` | (Mixin) Custon styles for dots | NULL

that you then `@apply` to the styles of the relative components.

This feature has been removed from browsers, so there is no way to edit the slider dots appearance. 

Could you change it in some other way?

Thank you

### PR DESCRIPTION
**Pull request by javier-godoy**
Close #41

### CODE REVIEW

File: src/main/resources/META-INF/resources/frontend/paper-slider/l2t-paper-slider.js [hunk 1]
```diff
@@ -1,665 +1,34 @@
 /*-
  * #%L
- * Carousel Addon
+ * Granite Alert
```
Comment (paodb) [inline src/main/resources/META-INF/resources/frontend/paper-slider/l2t-paper-slider.js:3]: Granite Alert?

**Review by paodb — CHANGES_REQUESTED** [review]

**Comment by paodb** [comment]
I don't think the original file lt2-paper-slider,js should be renamed (is the original file from the source library).

**Comment by javier-godoy** [comment]
@paodb I'm not sure how to name the web-components. (I think it's a breaking change either way)

**Review by paodb — APPROVED** [review]
LGTM

**Pull request merged**